### DISCUSSION TITLE
Editing checkout fields

### DISCUSSION THREAD

**Discussion by mathijsvdberg** [woocommerce/woocommerce-gutenberg-products-block#5483] [Ideas, Feature Requests and Feedback]
Hello,

How can I edit the checkout fiekds? I want to remove some and add some.

We don't offer technical support on GitHub so we recommend using the following:

## Reading our documentation
Usage docs can be found here: https://docs.woocommerce.com/

If you have a problem, you may want to start with the self help guide here: https://docs.woocommerce.com/document/woocommerce-self-service-guide/

**Technical support for premium extensions or if you're a WooCommerce.com customer**
 from a human being - submit a ticket via the helpdesk
https://woocommerce.com/contact-us/ 

## General usage and development questions
- WooCommerce Slack Community: https://woocommerce.com/community-slack/
- WordPress.org Forums: https://wordpress.org/support/plugin/woocommerce
- The WooCommerce Help and Share Facebook group

**Comment by nerrad** [comment]
Hi @MathijsCraft 👋🏻 

I'm assuming you are referring to editing checkout fields in the Checkout block. Currently there is no explicit interface for adding additional checkout fields or removing existing ones within the block. That is on our radar to add some interfaces (both user and code based) for achieving this functionality.

Since there's nothing actionable with this issue at the moment I'm going to close it, but I do appreciate you taking the time to leave the issue. It's noted that there is desire for this functionality.

**Comment by webdados** [comment]
No news on this? There are literally hundreds of plugins and customizations that need to add new fields to the checkout and that are unable to do so.
I would expect some kind of retro-compatibility with filters like `woocommerce_checkout_fields`, `woocommerce_billing_fields`, `woocommerce_shipping_fields`, and `woocommerce_default_address_fields`.

**Reply by passatgt** [reply]
I know it's not the best solution and i agree that we should have some filters to manage the fields, but until thats ready you can use a custom block to add a new field. Check out the [EU VAT Number](https://woocommerce.com/products/eu-vat-number/) extension as an example, it will create a new step to collect the VAT number.

**Reply by Stella3** [reply]
That's great if you actually deal with VAT numbers but I need something more simple. 

The ability to ask a custom question with a simple text box should exist. I always thought it was a bit much that it required a plugin like Checkout Field Editor for WooCommerce before but, unless I'm missing a plugin that is compatible with Blocks, it's just not possible at all now.

**Comment by opr** [comment]
Hi - closing this discussion in favour of #11174 where we will discuss the feature as we work on it. It'll be better to focus all communication in a single discussion, rather than the few we have open right now for this.

Thanks!
